app/routes/events/view/export.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return this.modelFor('events.view');
```

Comment:
Just give it a try. Remove the model method here. You should still be able to access the identifier at `model.identifier`